kubejs/server_scripts/constants/materials.js

```diff
@@ -0,0 +1,731 @@
+//priority: 1010
+
+// Used to determine which materials to unify
+const materialsToUnify = [
```

Comment:
I think we need to split these file into several new ones.

Reply:
Split this up. Left the materials file as just the 3 constants we use/used for unification. Can just drop the file if we think that almost unified will completely fill that need, however. 

Fixed up the metal/gem properties into a cleaner structure that should allow it to be more flexible. This should also replace the now remove `metals` constant as `metalProperties` has (or will have) a complete list of metals.